src/chains/filecoin/types/src/data-managers/manager.d.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference types="node" />
+import { LevelUp } from "levelup";
+import { SerializableObject, BaseConfig } from "../things/serializable-object";
+export declare type Instantiable<T> = {
+  new (...args: any[]): T;
+};
+export default class Manager<
+  T extends SerializableObject<C>,
+  C extends BaseConfig
+> {
+  #private;
```

Comment:
Does these do anything?

Reply:
No, but everything under `src/chains/filecoin/types` is autogenerated by `ttsc` with the `npm run tsc.filecoin.declarations` command